nixos/modules/services/web-apps/funkwhale.nix

```diff
@@ -0,0 +1,378 @@
+{ lib
+, config
+, pkgs
+, ...
+}:
+
+with lib;
+
+let
+  cfg = config.services.funkwhale;
+
+  manage = let
+    setupEnv = lib.concatStringsSep "\n" (mapAttrsToList (name: val: "export ${name}=\"${val}\"") cfg.settings);
+  in pkgs.writeShellScript "funkwhale" ''
+    ${setupEnv}
+    export PYTHONPATH="${cfg.package.pythonPath}"
+    test -e /var/lib/funkwhale/secret_key && {
+      export $(cat /var/lib/funkwhale/secret_key | xargs)
+    }
+    ${optionalString (cfg.secretSettingsFile != null) "source ${cfg.secretSettingsFile}"}
+      exec ${cfg.package}/bin/funkwhale "$@"
+  '';
```

Comment:
This currently gets installed into `/var/lib/funkwhale/bin/funkwhale`. It is required to import media as far as I can tell, so I wonder if we can somehow get it into PATH from inside this module.

Reply:
I think we would need to add it to systemPackages.